runtime/src/bank.rs

```diff
@@ -74,6 +74,7 @@ pub const SECONDS_PER_YEAR: f64 = 365.25 * 24.0 * 60.0 * 60.0;
 pub const MAX_LEADER_SCHEDULE_STAKES: Epoch = 5;
 
 type BankStatusCache = StatusCache<Result<()>>;
+#[frozen_abi(digest = "9chBcbXVJ4fK7uGgydQzam5aHipaAKFw6V4LDFpjbE4w")]
```

Comment:
I believe this is caused by #10748 (FYI: @mvines @jackcmay ) aed8c474fc2826e6904d2c75747e057d2b8b49bb